docs/src/contributing.md

```diff
@@ -117,3 +119,25 @@ Pkg.add("LiveServer") # this is necessary only the first time, to install LiveSe
 using LiveSever: serve
 serve(; dir="docs/build")
 ```
+
+### Draft documentation
+
+If you want to quickly build a draft copy of the documentation (i.e. without running the doctests or all the examples, which can be very time-consuming), set the environment variable `BREEZE_DOCS_DRAFT=true` and run again the `docs/make.jl` script:
+
+```sh
+export BREEZE_DOCS_DRAFT=true
```

Comment:
```suggestion
export BREEZE_DRAFT_DOCS=true
```

my only excuse being that you did write "draft version of the docs", so IO would say these are "draft docs" :-D :-D